libcloudforensics/providers/gcp/internal/storage.py

```diff
@@ -173,3 +175,63 @@ def DeleteObject(self, gcs_path: str) -> None:
     gcs_objects = self.GcsApi().objects()
     request = gcs_objects.delete(bucket=bucket, object=object_path)
     request.execute()  # type: Dict[str, Any]
+
+  def GetBucketSize(self,
+                    bucket_name: str = None,
+                    timeframe: int = 1) -> Dict[str, int]:
+    """List the size of a Google Storage Bucket in a project (default: last 1 day).
```

Comment:
size == total bytes stored, yes?

Reply:
Correct, I've added "(in bytes)" in some appropriate doc strings to make this clear